packages/eslint-config-vivy/index.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  env: {
+    browser: true,
+    node: true,
+    jest: true,
+    "jest/globals": true
+  },
+  parser: 'babel-eslint',
+  extends: [
+    'airbnb',
+    'prettier',
+    'prettier/react',
+    "plugin:jest/recommended"
+],
+  "plugins": [
+    "prettier",
+    "jest"
+  ],
+  rules: {
+    "class-methods-use-this": 0,
+    "import/prefer-default-export": 0,
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "prettier/prettier": ["error"],
+    "no-underscore-dangle": 0,
+    "import/no-extraneous-dependencies": false
```

Comment:
yeah agree. 
well this is there because we use it in storybook. 
Why we need it in storybook? 
When we bundle storybook components we dont actually need any `@storybook` dependency. Like for instance 
```
    "@storybook/addon-storyshots": "^5.0.6",
    "@storybook/addon-info": "^5.0.6",
    "@storybook/addon-storyshots-puppeteer": "^5.0.6",
    "@storybook/addons": "^5.0.6",
    "@storybook/react": "^5.0.6",
    "@storybook/storybook-deployer": "^2.8.1",
    "@storybook/theming": "^5.0.6"
```
We dont want that so that the bundle size of the library is actually small. 
In each story tho we have an import 
```
import { storiesOf } from "@storybook/react";
```
and when this is not activated eslint would show an error. 

We can ofc include this rule only on storybook if we would like.

Reply:
Ah, sounds like they should just be dev dependencies? 

But yeah I think either way this should then be specific only to our vivy-components config 👍